src/main/java/com/google/flourbot/api/Bot.java

```diff
@@ -67,26 +71,31 @@ public void onEvent(@RequestBody JsonNode event) throws IOException, GeneralSecu
           replyText = String.format("Thanks for adding me to a DM, %s!", displayName);
         }
         break;
+
       case "MESSAGE":
         // Sends request to execution module
         String email = event.at("/message/sender/email").asText();
         String message = event.at("/message/text").asText();
         String threadId = event.at("/message/thread/name").asText();
-        replyText = macroExecutionModule.execute(email, message, threadId);
+        
+        ChatResponse chatResponse = macroExecutionModule.execute(email, message, threadId);
+        replyText = chatResponse.getReplyText();
+        Card card = CardResponse.createCardResponse(replyText);
```

Comment:
Are we sure that we want everything to be a card, is a card with just a text the same as simple text response?

Reply:
Updated!